examples/todo/imports/ui/profile/ProfileContainer.js

```diff
@@ -19,11 +21,58 @@ export const editProfileMutation = graphql(
   }
 );
 
+
 const loggedUser = getLoggedUserContext();
 
+handleSubmitProfile = (editProfileState) => !!editProfileState.firstName.trim()
+
 export const ProfileContainer = compose(
   editProfileMutation,
   loggedUser,
   withRouter,
-  withApollo
+  withApollo,
+  // withAlert,
+  withState('editProfileState', 'setEditProfileState', props =>
+    intialState = {
+      _id: props.loggedUser._id,
+      email: props.loggedUser.email,
+      firstName: props.loggedUser.firstName || '',
+      lastName: props.loggedUser.lastName || '',
+      age: props.loggedUser.age || 0,
+    },
+  ),
+  withHandlers({
+    // eslint-disable-next-line no-undef
+    onInputChange: (props) => ({ target: { name, value } }) =>
+      props.setEditProfileState({ ...props.editProfileState, [name]: value }),
+
+    editProfileAndGo: (props) => () => {
+      if (this.handleSubmitProfile(props.editProfileState)) {
+        const { editProfile, history, client } = props;
+        const { firstName, lastName, age } = props.editProfileState;
+        editProfile({
+          variables: {
+            user: {
+              firstName, lastName, age
+            },
+          },
+        })
+          .then(() => {
+            // TODO when we have the cache working correctly after an update we can remove this
+            // https://github.com/CodeFTW/future-web/issues/34
+            client.resetStore();
+            // TODO: I don't know how can I show the alert for user
+            console.log('Profile updated');
+            history.push('/profile');
+          })
+          .catch(error => {
+            // eslint-disable-next-line no-console
+            console.log(error);
+          });
+      } else {
+         // TODO: I don't know how can I show the alert for user
```

Comment:
what is the problem with alert?

Reply:
old comment, I solved it with a update the package of ui-alert in 1.0.3.